samples/nrf9160/modem_shell/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -0,0 +1,72 @@
+#Overlay to use nRF7002EK on top of nrf9160DK for Wi-Fi scanning
+
+# Disable Modem traces as uart1 disabled
+CONFIG_NRF_MODEM_LIB_TRACE=n
+
+# Need to disable some MoSh features to have more SRAM
+CONFIG_MOSH_WORKER_THREADS=n
+CONFIG_MOSH_IPERF3=n
+CONFIG_MOSH_CURL=n
+CONFIG_MOSH_SOCK=n
+CONFIG_MOSH_SMS=n
+CONFIG_MOSH_STARTUP_CMDS=n
+CONFIG_MOSH_AT_CMD_MODE=n
+CONFIG_MOSH_FOTA=n
+CONFIG_SHELL_CMD_BUFF_SIZE=512
+
+CONFIG_LOCATION_METHOD_WIFI=y
+CONFIG_LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD=y
+
+CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT=60
+CONFIG_LOCATION_WORKER_STACK_SIZE=8192
+
+# Does not work with buttons&LEDs
+CONFIG_DK_LIBRARY=n
+
+# Not for LTE throughput testing
+CONFIG_NRF_MODEM_LIB_SHMEM_TX_SIZE=4096
+CONFIG_NRF_MODEM_LIB_SHMEM_RX_SIZE=4096
+
+# Actual configs for the Wi-Fi
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+CONFIG_NET_L2_WIFI_SHELL=y
+
+# System settings
+CONFIG_NEWLIB_LIBC=y
+
+# Scan only using offload API
+CONFIG_WPA_SUPP=n
+
+# In 9160 default is socket interface
+CONFIG_NET_DEFAULT_IF_ETHERNET=y
+CONFIG_MBEDTLS=n
+CONFIG_NORDIC_SECURITY_BACKEND=n
+
+# Networking
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_SHELL=y
+CONFIG_NET_NATIVE=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_STATISTICS=y
+CONFIG_NET_STATISTICS_WIFI=y
+CONFIG_NET_STATISTICS_USER_API=y
+CONFIG_NET_CONTEXT_SYNC_RECV=y
+
+# Disable unused networking options
+CONFIG_NET_IPV6=n
+
+# Memories
```

Comment:
Is "Memories" correct in this context? @peknis? Sounds a bit odd to me. :) Could be just "Memory" or "Memory configurations".

Reply:
Definitely not Memories.